### PR TITLE
feat: add base logic for component

### PR DESCRIPTION
**Pull request by Sashuteak**
Add base logic for component as existing in old version.

### CODE REVIEW

**Comment by josephsaffron** [comment]
you can run `npx eslint . --ext .js,.html --fix` to fix your lint errors automatically.  Looks like your line endings are being wrongly set by your IDE.

File: custom-my-courses-in-other-orgs.js [hunk 1]
```diff
@@ -1,54 +1,177 @@
...
+import '@brightspace-ui/core/components/list/list-item.js';
+import '@brightspace-ui/core/components/list/list-item-content.js';
+import '@brightspace-ui/core/components/loading-spinner/loading-spinner.js';
+
 import { css, html, LitElement } from 'lit-element';
-import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
+import { fetchMyCoursesInOtherOrgs } from './src/dataApiClient.js';
+import { LocalizeLitMixin } from './src/mixins/localize-lit-mixin.js';
+
+const NOTHING = html``;
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:13]: nit: you can probably just inline this

File: custom-my-courses-in-other-orgs.js [hunk 2]
```diff
@@ -1,54 +1,177 @@
...
+		}
+
+		return html`
+			<d2l-list-item>
+				<d2l-button @click=${this._loadMore}>
+					${this.loadingMoreItems ? this.localize('loading') : this.localize('loadMore') }
+				</d2l-button>
+			</d2l-list-item>`;
+	}
+	renderLoadSpiner() {
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:129]: typo
```suggestion
	renderLoadSpinner() {
```

File: custom-my-courses-in-other-orgs.js [hunk 3]
```diff
@@ -1,54 +1,177 @@
...
+				</div>
+			</div>`;
+	}
+
+	renderListItems() {
+		return this.myCoursesInOtherOrgsList.map((item) => {
+			return html`
+				<d2l-list-item>
+					<d2l-list-item-content>
+						<d2l-link href="/d2l/customization/mycoursesinotherorgs/GuidSSO?orgUnitId= ${item.OrgUnitId}; orgId= ${item.OrgId}" target="_blank">${item.OrgUnitName} (${item.OrgName})</d2l-link>
```
Comment (josephsaffron) [inline custom-my-courses-in-other-orgs.js:111]: the href part of this link doesn't really look right.

File: src/dataApiClient.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { d2lfetch } from 'd2l-fetch/src';
```
Comment (josephsaffron) [inline src/dataApiClient.js:1]: if you set up a clientFactory model like what CPD has here: https://github.com/Brightspace/continuous-professional-development/tree/master/src/services
you can include sample json so the demo will show items even when it's running alone.

File: custom-my-courses-in-other-orgs.js (hunk 3 above)
Comment (Sashuteak) [reply custom-my-courses-in-other-orgs.js:111]: can you give more detail? 
this solution works without problems

File: custom-my-courses-in-other-orgs.js (hunk 1 above)
Comment (Sashuteak) [reply custom-my-courses-in-other-orgs.js]: Done

File: src/dataApiClient.js (hunk 4 above)
Comment (Sashuteak) [reply src/dataApiClient.js]: Done

File: custom-my-courses-in-other-orgs.js (hunk 3 above)
Comment (josephsaffron) [reply custom-my-courses-in-other-orgs.js:125]: the link renders like this: 
image

and clicking it takes you to what looks like an invalid url. That's what I'm referring to.

Comment (Sashuteak) [reply custom-my-courses-in-other-orgs.js]: Done

**Review by josephsaffron — APPROVED** [review]

**Review by Bumpie7 — APPROVED** [review]

**Pull request merged**